src/postamble_minimal.js

```diff
@@ -26,28 +26,64 @@ function exitRuntime(ret) {
 }
 #endif
 
+{{{ globalThis.argc_argv = function(condition) {
+    if (!MAIN_READS_PARAMS) return '';
+    return `argc, ${MEMORY64 ? 'BigInt(argv)' : 'argv'}`;
```

Comment:
Can you do `argc, ${to64('argv')}` here?

Reply:
check